examples/roitberg/pyanitools.py

```diff
@@ -0,0 +1,126 @@
+# Written by Roman Zubatyuk and Justin S. Smith
+import h5py
+import numpy as np
+import platform
+import os
+
+PY_VERSION = int(platform.python_version().split('.')[0]) > 3
+
+
+class datapacker(object):
```

Comment:
This is vendor'd code by the roitberg group that I don't want to modify/fork. They didn't upload this onto pip so I'm including it here.